cellpack/autopack/Gradient.py

```diff
@@ -73,7 +73,17 @@ def __init__(self, name, mode="X", description="", direction=None, bb=None, **kw
             self.computeStartEnd()
         self.function = self.defaultFunction  # lambda ?
         self.weight = None
-        self.list_mode = ["X", "Y", "Z", "-X", "-Y", "-Z", "direction", "radial"]
+        self.available_modes = [
+            "X",
+            "Y",
+            "Z",
+            "-X",
+            "-Y",
+            "-Z",
+            "direction",
+            "radial",
+            "surface",
+        ]
```

Comment:
I wonder about these modes.... they could in general have associated parameters or something ("radius" for radial?). Could be worthwhile to turn these modes into objects with attributes at some point.